checkov/sast/record.py

```diff
@@ -68,23 +68,48 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
 
         severity_message = f'\tSeverity: {self.severity.name}\n' if self.severity and self.show_severity else ''
 
-        file_details = f'{self.file_path}:{" -> ".join([str(x) for x in self.file_line_range])}' if \
-            self.file_line_range[0] != self.file_line_range[-1] else \
-            f'{self.file_path}:{str(self.file_line_range[0])}'
-        code_lines = self.get_code_lines_string(self.code_block)
-        detail = self.get_details_string(self.details)
-        caller_file_details = self.get_caller_file_details_string(self.caller_file_path,
-                                                                  self.caller_file_line_range)
         evaluation_message = self.get_evaluation_string(self.evaluations, self.code_block)
 
-        status_message = colored("\t{} for file - {}\n".format(status, file_details), status_color)
-
         cwe_message = colored(f'\t{self.cwe}\n') if self.cwe else ''
 
+        if self.metadata is not None and self.metadata.taint_mode is not None and self.metadata.taint_mode.data_flow \
+                is not None and len(self.metadata.taint_mode.data_flow) > 0:
```

Comment:
```suggestion
        if self.metadata and self.metadata.taint_mode and self.metadata.taint_mode.data_flow  and self.metadata.taint_mode.data_flow:
```

Reply:
you wrote `self.metadata.taint_mode.data_flow  and self.metadata.taint_mode.data_flow:` twice